include/openspace/events/eventengine.h

```diff
@@ -44,6 +46,11 @@ class EventEngine {
         std::optional<ghoul::Dictionary> filter;
     };
 
+    struct TopicInfo {
+        size_t id;
```

Comment:
Should initialize this to some number, similar to the ActionInfo, so maybe:
```suggestion
        uint32_t id = std::numeric_limits<uint32_t>::max();
```
?